core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -147,37 +167,46 @@ public enum Property {
       "1.3.5"),
   @Sensitive
   INSTANCE_SECRET("instance.secret", "DEFAULT", PropertyType.STRING,
-      "A secret unique to a given instance that all servers must know in order"
-          + " to communicate with one another. It should be changed prior to the"
-          + " initialization of Accumulo. To change it after Accumulo has been"
-          + " initialized, use the ChangeSecret tool and then update accumulo.properties"
-          + " everywhere. Before using the ChangeSecret tool, make sure Accumulo is not"
-          + " running and you are logged in as the user that controls Accumulo files in"
-          + " HDFS. To use the ChangeSecret tool, run the command: `./bin/accumulo"
-          + " admin changeSecret`.",
+  // @formatter:off
+      """
+          A secret unique to a given instance that all servers must know in order \
+          to communicate with one another. It should be changed prior to the \
+          initialization of Accumulo. To change it after Accumulo has been \
+          initialized, use the ChangeSecret tool and then update accumulo.properties \
+          everywhere. Before using the ChangeSecret tool, make sure Accumulo is not \
+          running and you are logged in as the user that controls Accumulo files in \
+          HDFS. To use the ChangeSecret tool, run the command: `./bin/accumulo \
+          admin changeSecret`.
+      """, // @formatter:on
```

Comment:
Turning the formatter off and on seems excessive. As far as I can tell, this only prevents the final """ from aligning with the block which seems fine to allow. Maybe this is a consideration for the formatter plugin, which I believe @ctubbsii wrote, but not really important.

I have no problem with the switch to a multi-line string for these cases though, it is more legible since it aligns everything

Reply:
Will this block have leading white space? Reading this section`Incidental White Space` in these [docs](https://docs.oracle.com/en/java/javase/17/text-blocks/index.html) it seems like it might, but not sure.

Reply:
The formatter in Eclipse 2025-06 seems to work fine with these, with or without the trailing slash. It doesn't mangle them. But, it also doesn't give too many options for wrapping/reflow. It tends to keep what you have. It only has config for the default indentation.

I can release a new version of the formatter plugin, based on Eclipse 2025-06, if that is what is needed to get this working properly. That's not a big deal at all.